ui/tests/acceptance/setup.js

```diff
@@ -0,0 +1,11 @@
+import path from 'path'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['path' is defined but never used.](https://app.codacy.com/gh/owncloud/ocis-hello/pullRequest?prid=5709816)